### PR TITLE
Retro games list is now online! 🌏

### LINKED ISSUE #7: Host game list file online
In order to avoid updating the Firefox extension every time we want to add a new game title to the list of available games, it is better to **host the game_list.js file online** and not locally. [Another option would be to create an API online.]

We can do that either on an online hosting service or more preferably on Github's raw content viewer (e.g., https://raw.githubusercontent.com/KAUTH/Retro-Gamez/master/retro-gamez/popup/game_list.js?).

In both cases, it is important to include the [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) header in the host server's response to the browser.

In Github, it supposedly returns a CORS header automatically (needs to be checked).
On an Apache-like server we need to add the following code in the _.htaccess_ file:
`Header set Access-Control-Allow-Origin "*"`

### PR DESCRIPTION
**Pull request by paulopacitti**
## Description :pencil2:
- Changed `game_list.js` to `game_list.json`;
- Replaced `var` with `const` and `let` (best practices);
- Added `<meta charset...` for web compatibility;
- Now the games list is fetched via `GET` from GitHub, so it's not necessary anymore to release a new version of the browser extension everytime a new game is added to the list;
- The GitHub endpoint is pointed to my forked repo, so before merging this PR, make sure to mention me so I can change it or you can change by this endpoint yourself too :)

Closes #7

### CODE REVIEW

**Comment by paulopacitti** [comment]
@KAUTH waiting for your approval ;)

**Comment by KAUTH** [comment]
> @KAUTH waiting for your approval ;)

Hey thanks for the PR, I will review it ASAP! 😊

File: retro-gamez/popup/retro_gamez.html [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-        <iframe id="iframe" width="600" height="450" frameborder="no" allowfullscreen="true" webkitallowfullscreen="true" mozallowfullscreen="true" scrolling="no"></iframe>
-
-    </body>
-
-        <script src="game_list.js"></script>
-        <script src="retro_gamez.js"></script>
-
+  <head>
+    <meta charset="utf-8"/>
+    <title>RETRO GAMEZ</title>
```
Comment (KAUTH) [inline retro-gamez/popup/retro_gamez.html:5]: Change title to "Retro-Gamez"

File: retro-gamez/popup/game_list.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
-var gameList = [{
+[
```
Comment (KAUTH) [inline retro-gamez/popup/game_list.json:1]: Copy the latest list of games from the repo.

File: retro-gamez/popup/retro_gamez.js [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
+	const oneDay = 1000 * 60 * 60 * 24;
+	const day = Math.floor(diff / oneDay);
+
 	return day;
 }
 
-// Getting the array of objects containing the available games from a local JS file, stored in the gameList variable
-var games = gameList;
-var totalGames = Object.keys(games).length;
+async function main() {
```
Comment (KAUTH) [inline retro-gamez/popup/retro_gamez.js:15]: I don't believe it is necessary to have a main() function:

[Stack Overflow 1](https://stackoverflow.com/questions/9015836/why-doesnt-javascript-need-a-main-function#:~:text=Many%20programming%20languages%20require%20a,a%20function%20is%20not%20required.)
[Stack Overflow 2](https://stackoverflow.com/questions/8629859/main-function-in-javascript/8629865)

**Review by KAUTH — CHANGES_REQUESTED** [review]

File: retro-gamez/popup/retro_gamez.js (hunk 3 above)
Comment (KAUTH) [reply retro-gamez/popup/retro_gamez.js:15]: I just realized you have a main() function or a function for that matter in order to use async and wait.

[Stack Overflow](https://stackoverflow.com/questions/43832490/is-it-possible-to-use-await-without-async-in-js#:~:text=No.,sense%20in%20an%20async%20function.)

Comment (paulopacitti) [reply retro-gamez/popup/retro_gamez.js:15]: > I just realized you have a main() function or a function for that matter in order to use async and wait.
exactly hahahaha

**Comment by paulopacitti** [comment]
Resolved your requests! @KAUTH

**Review by KAUTH — APPROVED** [review]

**Pull request merged**